socket.c

```diff
@@ -78,6 +79,12 @@ bool cSatipSocket::Open(const int portP, const bool reuseP)
      ERROR_IF_FUNC(setsockopt(socketDescM, SOL_IP, IP_PKTINFO, &yes, sizeof(yes)) < 0,
                    "setsockopt(IP_PKTINFO)", Close(), return false);
 #endif // __FreeBSD__
+     // Tweak receive buffer size if requested
+     if (rcvBufSizeM > 0)
+     {
```

Comment:
this should be at the end of previous line

Reply:
Done.